Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -4861,6 +4861,18 @@ RefPtr<CSSValue> ComputedStyleExtractor::valueForPropertyInStyle(const RenderSty
         if (style.positionAnchor().isNull())
             return CSSPrimitiveValue::create(CSSValueAuto);
         return CSSPrimitiveValue::createCustomIdent(style.positionAnchor());
+    case CSSPropertyTimelineScope:
+        switch (style.timelineScope().type) {
+        case TimelineScope::Type::None:
+            return CSSPrimitiveValue::create(CSSValueNone);
+        case TimelineScope::Type::All:
+            return CSSPrimitiveValue::create(CSSValueAll);
+        case TimelineScope::Type::Ident:
+            return CSSPrimitiveValue::createCustomIdent(style.timelineScope().string);
+        default:
+            ASSERT_NOT_REACHED();
+            return CSSPrimitiveValue::create(CSSValueNone);
```

Comment:
No need for the `default` since you've covered all three cases above. You can always add the `ASSERT_NOT_REACHED()` and `return` after the `switch()`.

Reply:
Fixed.